doc/contributing/maintaining-openssl.md

```diff
@@ -3,10 +3,12 @@
 This document describes how to update `deps/openssl/`.
 
 If you need to provide updates across all active release lines you will
-currently need to generate three PRs as follows:
+currently need to generate four PRs as follows:
 
 * a PR for master which is generated following the instructions
-  below.
+  below for OpenSSL 3.0.x
+* a PR for 16.x which is generated following the instructions
+  below for OpenSSL 1.1.1
```

Comment:
Shouldn't we use the same wording as other branches?
```suggestion
* a PR for 16.x following the instructions in the v16.x-staging version
  of this guide.
```

Reply:
That probably makes sense although the 16.x branch has instructions for openssl 3.0 and 1.1.1 as well so we might want to update those at some point.